spp_farmer_registry_laos/views/event_data_view.xml

```diff
@@ -0,0 +1,322 @@
+<odoo>
+    <record id="view_spp_event_cycle_tree" model="ir.ui.view">
+        <field name="name">view_spp_event_cycle_tree</field>
+        <field name="model">spp.event.cycle</field>
+        <field name="priority">1</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="program_id" />
+                <field name="cycle_id" />
+                <field name="event_type" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_spp_event_cycle_form" model="ir.ui.view">
+        <field name="name">view_spp_event_cycle_form</field>
+        <field name="model">spp.event.cycle</field>
+        <field name="priority">200</field>
+        <field name="arch" type="xml">
+            <form string="Event Data: Farm" duplicate="0">
+                <sheet>
+                    <div class="row mt16 o_settings_container">
+                        <div class="col-4 o_setting_box">
```

Comment:
@emjay0921 can you add attribute `name` in this div that encapsulates the field? add this so that the other modules that will inherit this form view will have an easy way to add their fields using xpath.